### ISSUE TITLE
Missing mccs - 

### ISSUE THREAD

**Issue by AHAnalytics** [greggles/mcc-codes#23]
I have found following mccs which are not defined in your list. I've included the source of the details for each.
3303	TIGERAIR	https://usa.visa.com/content/dam/VCOM/download/merchants/visa-merchant-data-standards-manual.pdf
3308	China Southern: CSAIR	https://www.mastercard.us/content/dam/public/mastercardcom/na/global-site/documents/quick-reference-booklet-merchant.pdf
3839	Kyoritsu Hotels	https://usa.visa.com/content/dam/VCOM/download/merchants/visa-merchant-data-standards-manual.pdf
8082	Home Health Care Services	https://empower2.fisglobal.com/rs/092-EMI-875/images/Summer%202022%20NUNL.pdf
9751	UK Supermarkets, Electronic Hot File	https://www.citibank.com/tts/solutions/commercial-cards/assets/docs/govt/Merchant-Category-Codes.pdf
9752	UK Petrol Stations, Electronic Hot File	https://www.citibank.com/tts/solutions/commercial-cards/assets/docs/govt/Merchant-Category-Codes.pdf
7322	Debt Collection Agencies 	https://empower2.fisglobal.com/rs/092-EMI-875/images/Summer%202022%20NUNL.pdf

**Comment by quentingosset** [comment]
> I have found following mccs which are not defined in your list. I've included the source of the details for each. 3303 TIGERAIR https://usa.visa.com/content/dam/VCOM/download/merchants/visa-merchant-data-standards-manual.pdf 3308 China Southern: CSAIR https://www.mastercard.us/content/dam/public/mastercardcom/na/global-site/documents/quick-reference-booklet-merchant.pdf 3839 Kyoritsu Hotels https://usa.visa.com/content/dam/VCOM/download/merchants/visa-merchant-data-standards-manual.pdf 8082 Home Health Care Services https://empower2.fisglobal.com/rs/092-EMI-875/images/Summer%202022%20NUNL.pdf 9751 UK Supermarkets, Electronic Hot File https://www.citibank.com/tts/solutions/commercial-cards/assets/docs/govt/Merchant-Category-Codes.pdf 9752 UK Petrol Stations, Electronic Hot File https://www.citibank.com/tts/solutions/commercial-cards/assets/docs/govt/Merchant-Category-Codes.pdf 7322 Debt Collection Agencies https://empower2.fisglobal.com/rs/092-EMI-875/images/Summer%202022%20NUNL.pdf

if you find this official file : https://www.iso.org/standard/79450.html i can do the job to update the list.
@greggles are you still up on this repo ?

**Comment by greggles** [comment]
Yes, please post a pull request and review it and I can merge it.

**Comment by quentingosset** [comment]
> 

A friend and i have extract all the data (need fix some little bug) from VISA (April 2023) : https://usa.visa.com/content/dam/VCOM/download/merchants/visa-merchant-data-standards-manual.pdf

![image](https://github.com/greggles/mcc-codes/assets/6824137/4fc42f5a-d048-4208-a323-b8de9a261081)
![image](https://github.com/greggles/mcc-codes/assets/6824137/64cf5c12-e118-4e65-b79e-bc5bd16c0b67)

@greggles Do you have any ideas on how to arrange this in your core json? Do you have a proposal? 

Personally, I like the fact of having a general indication of the type of MCC as well as a more defined description.
The airline example. I like to know that my MCC is : Airlines and Air Carriers and that the description of my MCC corresponds to the airline.
